modules/transport/PostMessageTransport.js

```diff
@@ -0,0 +1,142 @@
+import Postis from 'postis';
+
+/**
+ * The list of methods of incomming postis messages that we have to support for
+ * backward compatability for the users that are directly sending messages to
+ * Jitsi Meet (without using external_api.js)
+ *
+ * @type {string[]}
+ */
+const oldIncommingMethods = [ 'display-name', 'toggle-audio', 'toggle-video',
```

Comment:
In general I don't like the "old" wording. I would prefer to call it legacyIncomingMethods. The "methods" wording is also confusing, because sometimes we use events, sometimes requests/responses in our jsdocs. See below.